packages/core/package.json

```diff
@@ -118,13 +118,13 @@
   "publishConfig": {
     "access": "public"
   },
-  "homepage": "https://github.com/vercel-labs/ai#readme",
+  "homepage": "https://vercel.com/templates/ai",
```

Comment:
@jaredpalmer Thoughts on the homepage here?

Reply:
```suggestion
  "homepage": "https://sdk.vercel.ai/docs",
```

Reply:
I'm going to change it to https://sdk.vercel.ai/docs which feels more appropriate

Reply:
Glad we both came to the same conclusion 😄